app/src/main/java/com/keylesspalace/tusky/network/AuthInterceptor.java

```diff
@@ -31,8 +47,16 @@ public Response intercept(@NonNull Chain chain) throws IOException {
         AccountEntity currentAccount = accountManager.getActiveAccount();
 
         Request.Builder builder = originalRequest.newBuilder();
+        // In the future we could add a phantom header parameter to some requests which would
+        // signalise that we should override current account (could be useful for "boost as.."
+        // actions and the like
         if (currentAccount != null) {
-            builder.header("Authorization", String.format("Bearer %s", currentAccount.getAccessToken()));
+            // I'm not sure it's enough the hostname but should be good
```

Comment:
yes, is enough (you are missing a verb in that comment but you can remove it if you want)